### DISCUSSION TITLE
Disable Header / Footer?

### DISCUSSION THREAD

**Discussion by gsbanner13** [OrchardCMS/OrchardCore#8254] [Q&A]
Hello!

This may be an odd question, but I was wondering...

I have the following structure implemented inside my Layout.Liquid page:

<body id="page-top" dir="{{ Culture.Dir }}">
	{% render_section "Header", required: false %}

	{% render_section "Content" %}

	{% render_section "Footer", required: false %} 
</body>

This works great, but I have been asked if there is a way to add a page that DOES NOT show the header and footer.  Is there a way to accomplish this?

**Comment by hishamco** [comment]
> This works great, but I have been asked if there is a way to add a page that DOES NOT show the header and footer. Is there a way to accomplish this?

I think this depend on your layout, you can create a layout without header and footer to be used in some pages and normal layout for the other pages
